test/parser/mixins.spec.js

```diff
@@ -25,6 +25,14 @@ describe('Parser', () => {
             expect(root.first.type).to.eql('atrule');
         });
 
+        it('should parse parametric mixins', () => {
+            const root = parse('.foo(@bar) { color: @bar;}', {mixinsAsAtRules: true, innerMixinsAsRules: true});
+
+            expect(root.first.type).to.eql('atrule');
+            expect(root.first.params[0].type).to.eql('atrule');
```

Comment:
eesh. disagree with that choice. adding `lessType` is just conflating what the syntax is supposed to handle. the entire purpose of a postcss syntax is to direct the parser on how to handle language-specific elements. all this does is tell the parser to continue on as normal while adding a tiny bit of metadata. 

also, `params` should contain the same `container` convenience methods, or be inserted as child nodes.